### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Euaalanlima** [langgenius/dify#7167] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

"I need your help. I’m having difficulties with Dify’s memory. For example, when I ask about lip fillers, it responds normally. However, if I then ask about the price of the lip filler, it seems to lose track of the information, as if I’ve started a new process and it doesn’t understand the context of the conversation. Ideally, when I mention a specific procedure at the beginning of the conversation, it should understand and maintain the context in subsequent questions. Could you help me with this?"

**Comment by patryk20120** [comment]
What type of app do you use?
